runtime/compiler/control/rossa.cpp

```diff
@@ -1236,19 +1236,42 @@ onLoadInternal(
 #endif
 
 #if defined(J9ZTPF)
+
+#if defined(J9VM_OPT_JITSERVER)
+   if (!(javaVM->internalVMFunctions->isJITServerEnabled(javaVM)))
+      {
```

Comment:
We can protect with `ifdef` only the `if` statement, without the curly bracket. Then we don't need to protect the ending curly bracked with another `ifdef`